cw_decred/lib/wallet_addresses.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:cw_core/wallet_addresses.dart';
+import 'package:cw_core/wallet_info.dart';
+import 'package:cw_core/address_info.dart';
+import 'package:cw_dcrlibwallet/cw_dcrlibwallet.dart';
+import 'package:mobx/mobx.dart';
+
+class WalletAddressesBase extends WalletAddresses {
```

Comment:
Maybe `DecredWalletAddresses`. Follows the pattern used by other coins.

Reply:
Do you think? Adding "Decred" to everything in the decred folder seemed redundant but the other coins do seem to do it too.

Reply:
Ah, true. I see you changed that pattern in this PR. I think for the file names, the `decred` prefix is an overkill, so good that they're gone. For the class names, I think it would help to have the prefix, so if/when they are used outside of the cw_decred package, they carry a specific meaning.

Reply:
yeah I guess if importing they do need Decred. Is everything exported from a file if you import it? Lower and upper case? Ill google.

Reply:
I think I've put Decred in front of all the extended classes now.